### PR TITLE
pija as a python package

### PR DESCRIPTION
**Pull request by llazzaro**
This will allow to install pija from pip with "pip install git+https://github.com/alcuadrado/pija.git".
Also installs all required deps (except opencv)

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+=======
+Credits
+=======
+
+Development Lead
+----------------
+
+* Patricio Palladino <email@patriciopalladino.com>
```
Comment (alcuadrado) [inline AUTHORS.rst]: There's also Matías Ré <aereal@gmail.com> @mattaereal.

**Comment by alcuadrado** [comment]
Thanks for this PR! I've always wanted to do this, but never took enough time.

I'll review your fist commit tomorrow. Did you use any tool to autogenerate all those files?

**Comment by llazzaro** [comment]
Yes, I used cookiecutter. let me know if you find something to change.

**Comment by mattaereal** [comment]
We never accepted this, though we were highly thankful for the work, because the package name will be changed for a more _inclusive_ term.

**Comment by mattaereal** [comment]
We will change the name, and if you're still interested on generating the package, feel free to do it. We'll let you know.

**Pull request closed without merging**